doc/user/content/serve-results/metabase.md

```diff
@@ -26,12 +29,20 @@ Database name     | **materialize**
 Database username | Materialize user.
 Database password | App-specific password.
 
-If you require SSL/TLS encryption, you can configure the connection to use `sslmode=require`. For more details, check out the [Metabase documentation](https://www.metabase.com/docs/latest/administration-guide/databases/postgresql.html).
+If you require SSL/TLS encryption, you can configure the connection to use
+`sslmode=require`. For more details and troubleshooting, check the
+[Metabase documentation](https://www.metabase.com/docs/latest/administration-guide/databases/postgresql.html).
```

Comment:
This phrasing doesn't make sense since Materialize went cloud native. Everyone is forced to use TLS encryption! We should just say "you must".